File: CoreJava/circle-app/src/com/techlabs/circle/Circle.java
package com.techlabs.circle;

public class Circle {
	private final float radius;
	private static final float pie = 3.14f;
	private BorderStyleOption borderstyle;

	public Circle(float rad) {
		radius = rad;
		borderstyle = BorderStyleOption.SINGLE;
	}

	public Circle(float rad, BorderStyleOption border_style) {
		radius = rad;
		borderstyle = border_style;
	}

	public float getRadius() {
		return radius;
	}

	public BorderStyleOption getBorderstyle() {
		return borderstyle;
	}

	public float calcArea() {
		return 2 * pie * getRadius();
	}

}

File: DotNET/C#/CircleApp/CircleApp/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CircleApp
{
    class Circle
    {
        private float _radius;
        private float _pie = 3.14f;
        private BorderStyle _borderstyle;

        public Circle(float rad)
        {
            _radius = rad;
            _borderstyle = BorderStyle.SINGLE;
        }

        public Circle(float rad, BorderStyle border_style)
        {
            _radius = rad;
            _borderstyle = border_style;
        }

        public float Radius
        {
            get
            {
                return _radius;
            }
        }

        public BorderStyle Borderstyle
        {
            get
            {
                return _borderstyle;
            }
        }

        public float calcArea()
        {
            return 2 * _pie * Radius;
        }

    }
}
